hw_diag/templates/diagnostics_page.html

```diff
@@ -87,7 +87,11 @@ <h3 class="text-center mb-4">Diagnostics Breakdown</h3>
                 </tr>
                 <tr>
                   <td>Frequency</td>
-                  <td class="text-right">{{ diagnostics.FR }}</td>
+                  {% if not diagnostics.FR %}
+                    <td class="text-right">Non Nebra miner</td>
```

Comment:
This seems overly specific. It's possible that Nebra miners in misconfigured fleets get caught here.

Reply:
Fair @marvinmarnold - got a better suggestion? Just don't want it to say "None" as this seems like an error when it's not.

Reply:
`Diagnostic not set` or `Not linked to diagnostics`? If the goal is to differentiate non-Nebra miners, we should create a new diagnostic (or deduce based on fleet name `BA`. Otherwise we can't determine whether or not something is an error just by frequency.

Reply:
`Diagnostic not set` or `Not linked to diagnostics`? If the goal is to differentiate non-Nebra miners, we should create a new diagnostic (or deduce based on fleet name `BA`. Otherwise we can't determine whether or not something is an error just by frequency.

Reply:
> @NebraLtd/developers @marvinmarnold any idea about why these tests are failing? I haven't changed anything to do with these?
> 
> ```
> FAILED hw_diag/tests/test_views_diagnostics.py::TestGetDiagnostics::test_add_gateway_txn_endpoint_success
> FAILED hw_diag/tests/test_views_diagnostics.py::TestGetDiagnostics::test_add_gateway_txn_failure_invalid_signature
> FAILED hw_diag/tests/test_views_diagnostics.py::TestGetDiagnostics::test_add_gateway_txn_failure_no_payload
> FAILED hw_diag/tests/test_views_diagnostics.py::TestGetDiagnostics::test_add_gateway_txn_success
> FAILED hw_diag/tests/test_views_diagnostics.py::TestGetDiagnostics::test_get_json_output
> FAILED hw_diag/tests/test_views_diagnostics.py::TestGetDiagnostics::test_version_endpoint
> ```

Just noticed this comment. Taking a look now, not obvious.

Reply:
Potentially: https://stackoverflow.com/questions/71661851/typeerror-init-got-an-unexpected-keyword-argument-as-tuple

Reply:
Yup, that fixed it: https://github.com/NebraLtd/hm-diag/tree/marvinmarnold/freq

Reply:
> `Diagnostic not set` or `Not linked to diagnostics`? If the goal is to differentiate non-Nebra miners, we should create a new diagnostic (or deduce based on fleet name `BA`. Otherwise we can't determine whether or not something is an error just by frequency.

I'm just more thinking for how this should display from a customer perspective. 

Because right now, on a rak miner, it just shows as frequency "None" on the diagnostics page and a few customers said that they thought that was an error.

Maybe if the Variant is non Nebra, we just hide frequency?

Reply:
Non-Nebra miner, you mean non-Nebra hardware generally? Because a RAK with in one of our official Balena RAK fleets should work correctly? That's why I was thinking we would base it off the fleet, not variant.

Reply:
@marvinmarnold with our balena rak fleet it is a single fleet - so they do not have any frequency environment variable set. So it just shows None

Another way we could do this is just have it set "Set by miner" when it's not available instead perhaps